lib/config.py

```diff
@@ -1,22 +1,11 @@
 import os
 
-AUTOGENERATED_CHANNELS = {'general': 'general',
-                          'logs': 'logs-📝', 'access': 'access-requests'}
-ARCHIVE_CATEGORY = 'ctf-archive'
-CTF_PREFIX = 'ctf '
-INTERNAL_CTFD_URL = 'http://localhost:8000'
-GUILD_IDS = list(map(int, os.getenv('GUILD_IDS').split(',')))
-DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
-CTFD_TOKEN = os.getenv('CTFD_TOKEN')
-DEFAULT_ARCHIVE_ID = 875120999132180521
-DEFAULT_FILE_ARCHIVE_ID = 1060700848633675807
-
-EMOTE_FROM = "<:pwnyPortalFrom:846826010561478657> "
-EMOTE_TO = "<:pwnyPortalTo:846826010241794102> "
-CTF_ROLE_ID = 881716982712528936
-UIUC_ROLE_ID = 613249703085342723
-HELPER_ROLE_ID = 616449707849285657
-ADMIN_ROLE_ID = 485530543836889098 # 987895918042964008
 
+DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD_IDS = os.getenv('GUILD_IDS', "").split(",")
+CTF_CATEGORY_CHANNELS = os.getenv("CTF_CATEGORY_CHANNELS", "").split(",")
+CTF_ROLES = os.getenv("CTF_ROLES", "").split(",")
```

Comment:
since these are all ENV vars, please provide an example ENV with these values so people can get up and running quickly.

Reply:
How should I change this beyond what is already in the readme?

Reply:
Never mind, you are right.